config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -218,22 +218,6 @@ presubmits:
         command:
         - ./hack/verify-gofmt.sh
 
-  - name: pull-test-infra-verify-govet
-    branches:
-    - master
-    always_run: true
-    decorate: true
-    path_alias: k8s.io/test-infra
-    labels:
-      preset-service-account: "true"
-    spec:
-      containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-experimental
-        command:
-        - runner.sh
-        args:
-        - ./hack/verify-govet.sh
```

Comment:
Can we just make `verify-govet.sh` use the nogo vet target? I don't see any reason to regress this script. We used this pattern for verify-deps.sh, which now uses `go mod`

Reply:
The way `nogo` works, the analyses are run at build time as part of `go build`, so there is no separate target to run _just_ the analyses, and similarly there is no way to build without running them. If we kept this script around it would just be something like `bazel build //...`

Reply:
Can we do that for at least one commit so that we don't break the presubmit?

Reply:
Sure.

Reply:
I did this, but it takes approximately forever

Might it be reasonable to just print a deprecation message and then pass?